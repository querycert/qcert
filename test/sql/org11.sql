select age, name
from students, employees
where eid = 1
  and sid = 1
;
