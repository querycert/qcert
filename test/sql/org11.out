[{"age" : { "$nat" : 26 },
  "name" : "Thandy Coe"}]